### PR TITLE
[TW-1068] Refresh Datadog

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
Refreshed Datadog integration page
- Verified steps and updated as needed
- Edited for grammar and style
- Updated screenshot
- Changed `updated`

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/datadog.md [hunk 1]
```diff
@@ -46,19 +43,22 @@ Setting up a Datadog integration requires you to get an API key from Datadog and
 
     <img alt="Home page and integrations" src="https://assets.postman.com/postman-docs/v10/home-integrations-v10.jpg" width="390px">
 
-1. Search and select **Datadog**.
+1. Select **Browse All Integrations**.
+1. Search and select **[Datadog](https://postman.postman.co/integrations/service/datadog)**.
 1. You can select **View All** for a list of all the integrations already created by your team.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/integrations/available-integrations/datadog.md:48]: What do you think about making this line a note under the previous step instead?

File: src/pages/docs/integrations/available-integrations/datadog.md [hunk 2]
```diff
@@ -46,19 +43,22 @@ Setting up a Datadog integration requires you to get an API key from Datadog and
...
-1. Select **Add Integration** to start the configuration process.
+1. Next to **Post monitoring results**, select **Add Integration** to start the configuration process.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/integrations/available-integrations/datadog.md:49]: Should we also mention the "Monitor API performance" integration?

File: src/pages/docs/integrations/available-integrations/datadog.md [hunk 3]
```diff
@@ -46,19 +43,22 @@ Setting up a Datadog integration requires you to get an API key from Datadog and
...
-   * Select the workspace and monitor whose data you would like to send to Datadog.
-   * Enter your [Datadog API key](https://docs.datadoghq.com/account_management/api-app-keys/).
-   * Select the Datadog region you would like to use.
-   * Optionally indicate if you want to send events for completed runs or send metrics for each run.
+    * **Nickname** - A nickname for your integration.
+    * **Workspace** - The workspace that has your monitor.
+    * **Monitor** - The collection-based monitor which will send its results to Datadog.
+    * **Region** - Select the Datadog region you want to use.
+    * **Datadog API Key** - Enter your [Datadog API key](#retrieving-your-datadog-api-key).
+    * Select the checkbox if you want to send events for completed urns.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/integrations/available-integrations/datadog.md:60]:
```suggestion
    * Select the checkbox if you want to send events for completed runs.
```

File: src/pages/docs/integrations/available-integrations/datadog.md [hunk 4]
```diff
@@ -70,11 +70,11 @@ When your monitor runs, the data starts flowing into Datadog.
 
 [![view in datadog](https://assets.postman.com/postman-docs/datadog-monitor-alerts.jpg)](https://assets.postman.com/postman-docs/datadog-monitor-alerts.jpg)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/integrations/available-integrations/datadog.md:71]:
```suggestion
![view in datadog](https://assets.postman.com/postman-docs/datadog-monitor-alerts.jpg)
```

Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/datadog.md:71]: It looks like the screenshots after this one also use the old format.

File: src/pages/docs/integrations/available-integrations/datadog.md [hunk 5]
```diff
@@ -23,7 +21,6 @@ contextual_links:
   - type: link
     name: "Postman Is Now a Datadog Technology Partner: More Integration Features"
     url: "https://blog.postman.com/postman-datadog-technology-partner-more-integration-features/"
-
 ---
 
 Datadog is a monitoring service for cloud-scale applications. It combines data from servers, databases, tools, and services to present a unified view of an entire stack. This integration allows you to configure your Postman Monitors to send metrics to Datadog where you can visualize and compare them with other metrics.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/integrations/available-integrations/datadog.md:26]: Should we add a "Contents" section?

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! I added some questions and suggestions. Thanks!

**Comment by akinard-postman** [comment]
Looks good to me! Just two questions:
- Do the tables' **Description** entries need periods?
- In the **Forwarding metrics** table, should GAUGE be non-code case?

File: src/pages/docs/integrations/available-integrations/datadog.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/integrations/available-integrations/datadog.md:49]: There's a note about this already at the end of the lede 😀

**Comment by mark-anthony-postman** [comment]
> Looks good to me! Just two questions:
> 
> * Do the tables' **Description** entries need periods?
> * In the **Forwarding metrics** table, should GAUGE be non-code case?

I think no on the first and yes on the second! 😀

File: src/pages/docs/integrations/available-integrations/datadog.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/datadog.md:60]: Yup I missed that. Sorry about that!

**Pull request merged**